src/java/main/org/apache/zookeeper/ClientCnxnSocket.java

```diff
@@ -223,4 +231,24 @@ abstract void doTransport(int waitTimeOut, List<Packet> pendingQueue,
      * finally unblock it when finished.
      */
     abstract void sendPacket(Packet p) throws IOException;
+
+    public ZKClientConfig getClientConfig() {
```

Comment:
Fixed, It is not required,removed it, made the instance variable clientConfig protected same as the other variable in the class